websites/A/Anim.wtf/metadata.json

```diff
@@ -1,16 +1,16 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.9",
 	"author": {
-		"name": "Luscarvalho",
-		"id": "458776046184169473"
+		"name": "nesi",
+		"id": "476522405725077519"
 	},
-	"service": "Puray Animes",
+	"service": "Anim",
```

Comment:
Folder name must match, the service name

Reply:
Renamed folder to Anim